src/components/Header/CKBNodeComp/CKBNodeModal.tsx

```diff
@@ -1,39 +1,171 @@
-import { useRef } from 'react'
+import { FC, PropsWithChildren, useRef, useState } from 'react'
 import { useTranslation } from 'react-i18next'
 import { useCKBNode } from '../../../hooks/useCKBNode'
 import CommonModal from '../../CommonModal'
 import { HelpTip } from '../../HelpTip'
 import { Switch } from '../../ui/Switch'
 import CloseIcon from '../../../assets/modal_close.png'
+import { ReactComponent as EditIcon } from './edit.svg'
+import { ReactComponent as DeleteIcon } from './delete.svg'
+
+import BackIcon from './back.png'
 import styles from './style.module.scss'
 
+const NodeEditForm: FC<
+  PropsWithChildren<{
+    initialData?: { alias: string; url: string }
+    onSubmit: (data: { alias: string; url: string }) => void
+  }>
+> = ({ children, initialData, onSubmit }) => {
+  const [alias, setAlias] = useState(initialData?.alias ?? '')
+  const [url, setUrl] = useState(initialData?.url ?? '')
+
+  return (
+    <form
+      className={styles.nodeEditForm}
+      onSubmit={e => {
+        e.preventDefault()
+        e.stopPropagation()
+        onSubmit({ alias, url })
+      }}
+    >
+      <div>
+        <label>Alias</label>
+        <input value={alias} onChange={e => setAlias(e.target.value)} />
```

Comment:
It looks like that the `state` is unnecessary and `input` can be an uncontrolled component.

`alias` and `url` can be fetched from the `form event` as

```ts
onSubmit = (e) => {
  const alias = e.currentTarget.alias.value
  const url = e.currentTarget.url.value
}
```